src/Main.java

```diff
@@ -12,33 +9,4 @@ public static void main(String[] args) {
 
 	}
 
-	public static void PlayAudio() {
-
-		if (Interface.dropDown.getSelectedItem() == "Montana") {
-			File Sound = new File("scarface_montana.wav");
-			PlaySound(Sound);
-
-		} else {
-			File Sound1 = new File("scarface_say_hello.wav");
-			PlaySound(Sound1);
-		}
-
-	}
-
-	static void PlaySound(File Sound) {
-
-		try {
-			File file = new File("Sound");
-			Clip clip = AudioSystem.getClip();
-			clip.open(AudioSystem.getAudioInputStream(Sound));
-			clip.setFramePosition(0);
-			clip.start();
-
-
-		} catch (Exception e) {
-			e.printStackTrace();
-
-		}
-	}
-
```

Comment:
This void should be moved to MassageListener class. I did this and changed it a little bit.